internal/storage/index/index.go

```diff
@@ -329,22 +331,17 @@ func (idx *index) Inspect() map[string]meta {
 	return entries
 }
 
-func (idx *index) GetPolicies(_ context.Context) ([]*policy.Wrapper, error) {
+func (idx *index) ListPolicyIDs(_ context.Context) ([]string, error) {
 	idx.mu.RLock()
 	defer idx.mu.RUnlock()
 
-	entries := make([]*policy.Wrapper, 0)
-	for _, modID := range idx.fileToModID {
-		pol, err := idx.loadPolicy(modID)
-		if err != nil {
-			return nil, err
-		}
-
-		wp := policy.Wrap(pol)
-		entries = append(entries, &wp)
+	files := make([]string, 0, len(idx.modIDToFile))
+	for _, f := range idx.modIDToFile {
+		files = append(files, f)
 	}
```

Comment:
Better to use indexing rather than append here. See my comment below.

Reply:
Ah, it's a map. We could have a manually incremented index variable but I am not too bothered about it.